man/docker-set.1.md

```diff
@@ -0,0 +1,78 @@
+% DOCKER(1) Docker User Manuals
```

Comment:
@hqhq Please reuse the text (*not* the metadata :-) from the `set.md` info above.